test/journal-route-test.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+const Journal = require('../model/journal.js');
+const PORT = process.env.PORT || 3000;
+
+// process.env.MONGODB_URI = 'mongodb://localhost/journaltest';
+
+require('../server.js');
+
+const url = `http://localhost:${PORT}`;
+const exampleJournal = {
+  name: 'test journal name',
+  subject: 'test journal subject'
+
+};
+
+
+describe('Journal Routes', function() {
+  describe('POST: /api/journal', function() {
+    describe('with a valid body', function(){
+      after( done => {
+        if (this.tempJournal) {
+          Journal.remove({})
+          .then( () => done())
+          .catch(done);
+          return;
+        }
+        done();
+      });
+
+      it('should return a journal', done => {
+        request.post(`${url}/api/journal`)
+        .send(exampleJournal)
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.name).to.equal('test journal name');
+          expect(res.body.subject).to.equal('test journal subject');
+          this.tempJournal = res.body;
+          done();
+        });
+      });
+    });
+
+    describe('with an invalid body or none provided', function(){
+      it('should return a 400 error', done => {
```

Comment:
This should now be 'should return a 404 error' because of the explicit error messages we made to catch errors.